src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4337,12 +4337,19 @@ export class StudyViewPageStore
         onError: () => {},
         onResult: unknownIds => {
             if (unknownIds.length > 0) {
-                this.pageStatusMessages['unknownIds'] = {
-                    status: 'danger',
-                    message: `Unknown/Unauthorized ${
-                        unknownIds.length > 1 ? 'studies' : 'study'
-                    } ${unknownIds.join(', ')}`,
-                };
+                // if all studies are unknown, set flag for error page message
```

Comment:
i think this whole onResult handler is unnecessary.  study page should just be able to check the promise itself and if the the unknownOrUnauthorizedStudyIds is true, then it can return an <ErrorPage> (or embed an alert bar if there are some legit study idds)